tests/components/airnow/test_config_flow.py

```diff
@@ -142,12 +142,18 @@ async def test_options_flow(hass: HomeAssistant, setup_airnow) -> None:
     assert result["type"] == data_entry_flow.FlowResultType.FORM
     assert result["step_id"] == "init"
 
-    result = await hass.config_entries.options.async_configure(
-        result["flow_id"],
-        user_input={CONF_RADIUS: 25},
-    )
+    with patch(
```

Comment:
Probably patch already when setting up the config entry.

Reply:
I don't agree, by patching here we test that the reload on option change works.

Reply:
There are api calls during config entry setup. The patch path in the fixture looks questionable.

Reply:
Right, I'll double check the API calls are correct 👍